lang/en/cpdlogbook.php

```diff
@@ -21,3 +21,5 @@
 $string['pluginadministration'] = 'Administration';
 $string['name'] = 'Name';
 $string['points'] = 'Points';
+$string['entryname'] = 'Name';
+$string['createtitle'] = 'Create a New Entry';
```

Comment:
lang strings should only have captials for the first word, unless there are proper nouns in the string. So little 'n' for new